jetbrains-core/src/software/aws/toolkits/jetbrains/settings/AwsSettingsConfigurable.java

```diff
@@ -17,10 +19,13 @@
 import java.util.Objects;
 import javax.swing.JComponent;
 import javax.swing.JPanel;
+import javax.swing.JTextField;
 import org.jetbrains.annotations.Nls;
 import org.jetbrains.annotations.Nls.Capitalization;
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
+import software.aws.toolkits.jetbrains.services.lambda.execution.sam.SamInitRunner;
+import software.aws.toolkits.jetbrains.ui.wizard.SamInitProjectBuilderCommonKt;
```

Comment:
We can give our Kotlin files prettier names so that you don't have to have the "Kt" at the end - just add `@file:JvmName("WhateverYouWantItToBeCalledInJava")` to the top of the file.

Reply:
Nice to know! fixed.